### ISSUE TITLE
Share the Chromium instance across all installed Electron apps

### ISSUE THREAD

**Issue by Jop-V** [electron/electron#6210]
Currently, every Electron app bundles their own instance of Chromium. If Chromium (or Google Chrome perhaps?) is already present on the system, the bundled instance should be discarded and the installed one should be used instead. If not, the bundled instance should be installed on the system so that future Electron apps can discard theirs. This would save a lot of storage space for people that have a lot of Electron apps installed.

**Comment by baconface** [comment]
I also believe Chromium and possibly Node.js are patched by the Electron team. So they wouldn't be able to piggy back off of preinstalled instances anyways. Hence the reason for the runtime idea.

**Comment by alexchandel** [comment]
That discussion has diverged a little into a full-blown runtime.  It would be nice to separately track the ability to simply share Chromium framework globally (i.e. as a shared framework on macOS).

**Comment by towerbat** [comment]
yeahh

**Comment by armanyener** [comment]
I can see the appeal of having an isolated Chromium instance -with the desired version of Chromium- for the specific software from the developers perspective, but I think using shared Chromium instance across various Electron apps wouldn't break the majority of the apps written with it. Implementing this kind of a feature in build time would be such a disk oriented optimization like the way Tauri and Wails projects handle this matter.

**Comment by towerbat** [comment]
Yes. Chromium resources can be placed as a dependency on systems. And apps would just use it.
